frontend/server/src/main/java/com/amazonaws/ml/mms/http/HttpRequestHandler.java

```diff
@@ -127,7 +129,22 @@ private void handleListModels(ChannelHandlerContext ctx, FullHttpRequest req) {
     }
 
     private void handlePing(ChannelHandlerContext ctx) {
-        NettyUtils.sendJsonResponse(ctx, new StatusResponse("healthy"));
+        Map<String, String> status = new HashMap<>();
+        Map<String, Model> models = ModelManager.getInstance().getModels();
+        HttpResponseStatus ret = HttpResponseStatus.OK;
+        for (Map.Entry<String, Model> m : models.entrySet()) {
+            int numScaled = m.getValue().getMinWorkers();
+            int numWorking = m.getValue().getCurrentWorkers().get();
+            String s = numScaled + "/" + numWorking;
+            if (numScaled > 0) {
+                ret = HttpResponseStatus.NOT_FOUND;
+                if (numWorking > 0) {
+                    ret = HttpResponseStatus.OK;
+                }
+            }
+            status.put(m.getKey(), s);
+        }
+        NettyUtils.sendJsonResponse(ctx, new StatusResponse(JsonUtils.GSON.toJson(status)), ret);
```

Comment:
The response is always 200, so the checking of models status doesn't add any value to this API.